graph_def_editor/select.py

```diff
@@ -358,7 +358,7 @@ def get_within_boundary_ops(ops,
 
   Args:
     ops: an object convertible to a list of `gde.Node`. those ops define the
-      set in which to perform the operation (if a `tf.Graph` is given, it
+      set in which to perform the operation (if a `gde.Graph` is given, it
```

Comment:
Oops, I must have missed these references from the previous PR, thanks for fixing it up!